orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/target/HostAndRoleTarget.java

```diff
@@ -29,11 +30,32 @@ public String getType() {
         return "compound";
     }
 
-        @Override
+    @Override
     public String toString() {
         return "HostAndRoleTarget{" +
                 "role=" + role +
                 ",hosts=" + hosts +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Not so important, but equality could also receive tests.

Reply:
added some